ramlfications/errors.py

```diff
@@ -57,9 +57,32 @@ class LoadRAMLError(Exception):
 # Update MIME Media Type Exception
 #####
 
-class MediaTypeError(Exception):
+class MediaTypeError(BaseRAMLError):
     pass
 
 
-class InvalidVersionError(Exception):
+class InvalidVersionError(BaseRAMLError):
     pass
+
+
+class UnknownDataTypeError(BaseRAMLError):
+    pass
+
+
+class DataTypeValidationError(BaseRAMLError):
+    """A common validator type for data type validation errors
+
+    :param list position_hint: path in the validated object where the
+        error happens. can be None.
+    :param <any> value: value of the object which had a problem:
+        repl of this object will be present in the message
+    :param <any> message: message to provide to the user
+    """
+    def __init__(self, position_hint, value, message):
+        self.position_hint = position_hint
+        self.value = value
+        if position_hint is None:
+            position_hint = []
+        BaseRAMLError.__init__(self, "{0}: {1}, but got: {2}".format(
```

Comment:
Hmm I can't remember if this is old-style; what's beneficial about calling the parent class's `__init__` rather than something like `super(DataTypeValidationError, self).__init__(...)`?